core/00_primordials.js

```diff
@@ -613,4 +613,6 @@
   // Provide bootstrap namespace
   globalThis.__bootstrap ??= {};
   globalThis.__bootstrap.primordials = primordials;
+  globalThis.__bootstrap.internals ??= {};
+  globalThis.__bootstrap.internals.primordials = primordials;
```

Comment:
Ideally we shouldn't need this, figure out a way how we can still access primordials.